ql/src/java/org/apache/hadoop/hive/ql/log/LogDivertAppender.java

```diff
@@ -242,12 +244,17 @@ public static void registerRoutingAppender(org.apache.hadoop.conf.Configuration
     LoggerContext context = (LoggerContext) LogManager.getContext(false);
     Configuration configuration = context.getConfiguration();
 
+    PurgePolicy purgePolicy = IdlePurgePolicy.createPurgePolicy("60", null, "SECONDS", configuration);
```

Comment:
IIRC the 60s was configurable from the log4j2.properties earlier. Can we make this configurable via some means (preferably log4j2 properties)?